src/Fixer/ClassNotation/ClassDefinitionFixer.php

```diff
@@ -150,6 +154,10 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                 ->setAllowedTypes(['bool'])
                 ->setDefault(false)
                 ->getOption(),
+            (new FixerOptionBuilder('fix_constructor_arguments', 'Whether constructor argument list in anonymous classes should be single line.'))
```

Comment:
option name doesn't match it's functionality

Reply:
What would you suggest? Would `single_line_constructor_arguments` be better?

Reply:
@jrmajor `inline_constructor_arguments`